site/snippets/enhanced-apis-example/viem-client/nft/types-enums.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * The following types are taken from the alchemy-sdk package, with slight modifications for sake of example.
+ *
+ * @see: https://github.com/alchemyplatform/alchemy-sdk-js/blob/main/src/types/types.ts
+ */
+
+export enum NftTokenType {
+  ERC721 = "ERC721",
+  ERC1155 = "ERC1155",
+  NO_SUPPORTED_NFT_STANDARD = "NO_SUPPORTED_NFT_STANDARD",
+  NOT_A_CONTRACT = "NOT_A_CONTRACT",
+  UNKNOWN = "UNKNOWN",
+}
+
+export enum OpenSeaSafelistRequestStatus {
```

Comment:
```suggestion
export enum OpenseaSafelistRequestStatus {
```

Reply:
`OpenSea` is actually in capital case in the alchemy sdk (and on their website) :)